src/cljs/athens/views/left_sidebar.cljs

```diff
@@ -141,35 +147,40 @@
                                             (and (= source-order
                                                     (dec order))
                                                  (= @drag :above)) nil
-                                            (= @drag :below)       (dispatch [:left-sidebar/drop source-order order :after])
-                                            :else                  (dispatch [:left-sidebar/drop source-order order :before])))
+                                            (= @drag :below)       (rf/dispatch [:left-sidebar/drop source-order order :after])
+                                            :else                  (rf/dispatch [:left-sidebar/drop source-order order :before])))
                                         (reset! drag nil))})
         title]])))
 
 
 (defn left-sidebar
   []
-  (let [open? (subscribe [:left-sidebar/open])
+  (let [open?     (rf/subscribe [:left-sidebar/open])
         shortcuts (reactive/get-reactive-shortcuts)]
-    ;; (when @open?
-
-    ;; IF EXPANDED
-    [:div (use-style left-sidebar-style {:class (if @open? "is-open" "is-closed")})
-     [:div (use-style left-sidebar-content-style {:class (if @open? "is-open" "is-closed")})
-
-      ;; SHORTCUTS
-      [:ol (use-style shortcuts-list-style)
-       [:h2 (use-sub-style shortcuts-list-style :heading) "Shortcuts"]
-       (doall
-         (for [sh shortcuts]
-           ^{:key (str "left-sidebar-" (second sh))}
-           [shortcut-component sh]))]
-
-      ;; LOGO + BOTTOM BUTTONS
-      [:footer (use-sub-style left-sidebar-style :footer)
-       [:a (use-style notional-logotype-style {:href "https://github.com/athensresearch/athens/issues/new/choose" :target "_blank"}) "Athens"]
-       [:h5 (use-style {:align-self "center"})
-        [:a (use-style version-style {:href "https://github.com/athensresearch/athens/blob/master/CHANGELOG.md"
-                                      :target "_blank"})
-         (athens.util/athens-version)]]]]]))
+    (fn []
```

Comment:
why wrap all the hiccup in a `fn`?

Reply:
so it's a clear distinction between setup and rendering fn, returning fn from component is making this fn a rendering fn.